app/controllers/discovered_hosts_controller.rb

```diff
@@ -247,7 +247,7 @@ def find_by_name
   def find_multiple
     # Lets search by name or id and make sure one of them exists first
     if params[:host_names].present? or params[:host_ids].present?
-      @hosts = Host::Discovered.where("id IN (?) or name IN (?)", params[:host_ids], params[:host_names] )
+      @hosts = Host::Discovered.includes(:model, :fact_values, :interfaces).where("id IN (?) or name IN (?)", params[:host_ids], params[:host_names] )
```

Comment:
model, fact_values and interfaces where added for submit_multiple_destroy
multiple_delete complained about: model, location, organization.
When adding this, multiple_delete complains about unused eager loading:
```
Unused Eager Loading detected
  Host::Discovered => [:fact_values, :interfaces]
  Remove from your finder: :includes => [:fact_values, :interfaces]
```
Adding locations and organizations will cause the same warning for submit_multiple_destroy since they both use `find_multiple`.

Should I add everything and get unused eager loading? or just `model` which they both use? split this into two methods? check what the action is?